Library/Homebrew/formula.rb

```diff
@@ -2038,11 +2057,16 @@ def needs(*standards)
     #
     # The test will fail if it returns false, or if an exception is raised.
     # Failed assertions and failed `system` commands will raise exceptions.
-
     def test(&block)
       define_method(:test, &block)
     end
 
+    # TODO
+    def pour_bottle?(reason, &block)
+      @pour_bottle_reason = reason
+      @pour_bottle_block = block
```

Comment:
I think there should always be a reason. I cannot think of a situation where we would want to omit it and which default reason might apply then. But if there's a good example, I'd like to hear it.